app/assets/stylesheets/_landing-page.scss

```diff
@@ -14,12 +14,21 @@
     &:last-child {
       margin-bottom: 2rem;
     }
+
+    a {
+      color: white;
+    }
   }
 
   h3 {
     font-size: $base-font-size * 2;
   }
 
+  .explanation {
+    @include shift(2);
+    @include span-columns(8);
```

Comment:
I decided on 8 columns instead of 10 because it's narrow and easier to read (fewer characters per line) that way.